src/index.html

```diff
@@ -4,9 +4,318 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home" class="header__logo">
+        <img src="../src/images/logo.png" alt="logo-moyo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__ul">
```

Comment:
nav should be different BEM block

Reply:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
```
![image](https://user-images.githubusercontent.com/97547824/201751740-abe2adaf-7687-4351-aa23-78710a72f0e2.png)